include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -45,12 +45,12 @@ namespace SuperiorMySqlpp { namespace Converters
         struct ToIntegerParserImpl<T, 1, validate>
         {
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
-            static inline void call(T& result, const char*& str) noexcept(validate)
+            static inline void call(T& result, const char*& str) noexcept(!validate)
             {
                 char c = *str;
                 if (validate)
                 {
-                    if (c < '0' || c > '9')
+                    if (!(c>= '0' && c<= '9')) //This is sometimes faster then using ||
```

Comment:
Please add spaces after `c` variable and after beginning of the comment.

Reply:
ok